Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -303,6 +303,17 @@ class CredentialsManagerSpec: QuickSpec {
                 }
             }
             
+            context("custom keychain", closure: {
+                beforeEach {
+                    storage = A0SimpleKeychain(service: "test_service")
+                    credentialsManager = CredentialsManager(authentication: authentication,
+                                                            storage: storage ?? A0SimpleKeychain())
+                }
+                
+                it("should accept custom keychains") {
+                    expect(storage?.service).to(match("test_service"))
```

Comment:
This only tests that you set the service name in the service you previously created.  I don't think I explained this at all in the last comment.  
What we need to do here is set the storage object as you do. Then perform some kind of action. For example you can manually add something to the keychain instance and then call on the credentials manager to clear it, then test it's empty.